### PR TITLE
Create Triage Party yaml for jib 

### PR DESCRIPTION
**Pull request by mpeddada1**
Usage instructions: 
* Steps to render the triage-party page are provided in https://github.com/google/triage-party#try-it.
* The only difference is that we will need to run the following command for this yaml file:
```export GITHUB_TOKEN="$(cat $HOME/.github-token)"
docker build --tag=tp .
docker run \
  --rm \
  -e GITHUB_TOKEN \
  -v "$PWD/jib_triage_party.yaml:/app/config/config.yaml" \
  -p 8080:8080 tp```

### CODE REVIEW

File: jib_triage_party.yaml [hunk 1]
```diff
@@ -0,0 +1,754 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+settings:
+  name: jib
+  min_similarity: 0.8
+  repos:
+    - https://github.com/GoogleContainerTools/jib
```
Comment (loosebazooka) [inline jib_triage_party.yaml:19]: The only strange thing I feel about this is that the repos unrelated to jib are included in here. Should this config not reside in github?

**Review by loosebazooka — APPROVED** [review]
Lets just get this in so we can get started. We can update the config as we move on.

File: jib_triage_party.yaml (hunk 1 above)
Comment (mpeddada1) [reply jib_triage_party.yaml:19]: hm yeah I feel a little strange about this as well.

**Comment by mpeddada1** [comment]
Moving this file out of github.

**Pull request closed without merging**